README.md

```diff
@@ -177,6 +177,8 @@ Custom iOS views can be exposed by subclassing `RCTViewManager`, implementing a
 
 @implementation MyCustomViewManager
 
+RCT_EXPORT_MODULE();
```

Comment:
Incredibly small nit - the convention throughout the product leaves out the semicolon after using these macros, could we remove that?